test/tools/yulopti.cpp

```diff
@@ -171,28 +208,118 @@ class YulOpti
 		}
 	}
 
+	void objectApply(shared_ptr<Object> _object, function<void(Object&)> _fn)
```

Comment:
Could we come up with a more descriptive name for this function? Apply what to object?

Reply:
Renamed to `objectApplyFn`. Not sure if that's an improvement or not. Couldn't come up with anything more appropriate since it's a pretty generic function.

Reply:
I think `objectApplyFunction` would be better.

Reply:
Renamed to `objectApplyFunction`